documentation/docs/install-pmm/install-pmm-client/connect-database/postgresql.md

```diff
@@ -152,140 +158,214 @@ To configure the extension:
 
-    !!! note alert alert-primary ""
-        See [`pg_stat_monitor` online documentation](https://docs.percona.com/pg-stat-monitor/configuration.html) for details about available parameters.
+    !!! tip "Additional settings"
+        View all available settings with:
+        ```sql
+        SELECT * FROM pg_stat_monitor_settings;
+        ```
+        
+    For complete parameter documentation, see the [`pg_stat_monitor` online documentation](https://docs.percona.com/pg-stat-monitor/configuration.html).
 
-3. Start or restart your PostgreSQL instance. The extension starts capturing statistics from every database.
+3. Restart your PostgreSQL instance to activate the extension.
 
-4. In a `psql` session:
+4. Create the extension in a psql session to create the necessary views that allow you to access the collected statistics:
 
     ```sql
-    CREATE EXTENSION pg_stat_monitor;
-    ```
-    
-    This command creates the view where you can access the collected statistics.
-
-    !!! note alert alert-primary ""
-        We recommend that you create the extension for the `postgres` database. In this case, you receive the access to the statistics, collected from every database.
+    psql -d postgres -c "CREATE EXTENSION pg_stat_monitor;"
+     ```
 
+    !!! note "Best practice"
+    Create the extension in the postgres database to access statistics collected from all databases in your instance. This provides comprehensive monitoring without needing to enable the extension in each individual database.
 
-5. Check the version.
+5. Verify the extension installation:
 
     ```sql
     SELECT pg_stat_monitor_version();
     ```
 
-## Add service
+### Bucket-based data aggregation in `pg_stat_monitor`
+
+`pg_stat_monitor` collects and aggregates PostgreSQL statistics using a data structure called a bucket. Multiple buckets are linked together to form a bucket chain.
+
+Each bucket:
+
+- collects data for a specific time period
+- has a defined memory allocation
+- stores aggregated statistics until expiration or overflow
+
+#### Bucket lifecycle
+
+When a bucket's expiration time is reached, accumulated statistics are reset and data is stored in the next available bucket in the chain.
+
+When all buckets in the chain have been used, the first bucket is reused and its contents are overwritten.
 
-When you have configured your database server, you can add a PostgreSQL service with the user interface or on the command line.
+If a bucket fills before its expiration time is reached, data is discarded.
 
-### With the user interface
+## Add service to PMM
 
-To add the service With the user interface:
+After configuring your database server with the appropriate extension, you need to add it as a service to PMM. You can do this either through the PMM user interface or via the command line.
+
+=== "Using the PMM user interface"
```

Comment:
Something went wrong here as well. CLI and UI presented as completely separate tab items